mkmsdk/MKMClient.py

```diff
@@ -0,0 +1,17 @@
+from oauthlib.oauth1 import Client
+
+
```

Comment:
the documentation of this module is missing. Add a docstring and some comments, especially "why you should append ``oauth_token`` to the params list.